rocketpool-cli/service/service.go

```diff
@@ -975,28 +975,26 @@ func pruneExecutionClient(c *cli.Context) error {
 		fmt.Println("You are using Native Mode.\nThe Smart Node cannot prune your Execution client for you, you'll have to do it manually.")
 	}
 	selectedEc := cfg.ExecutionClient.Value.(cfgtypes.ExecutionClient)
-	switch selectedEc {
-	case cfgtypes.ExecutionClient_Besu:
-		if cfg.Besu.ArchiveMode.Value == true {
-			fmt.Println("You are using Besu as an archive node.\nArchive nodes should not be pruned. Aborting.")
-			return nil
-		}
+
+	// Don't prune besu if it's in archive mode
+	if selectedEc == cfgtypes.ExecutionClient_Besu && cfg.Besu.ArchiveMode.Value == true {
+		fmt.Println("You are using Besu as an archive node.\nArchive nodes should not be pruned. Aborting.")
+		return nil
 	}
 
-	if selectedEc == cfgtypes.ExecutionClient_Geth || selectedEc == cfgtypes.ExecutionClient_Besu {
-		if selectedEc == cfgtypes.ExecutionClient_Geth {
-			fmt.Printf("%sGeth has a new feature that renders pruning obsolete. However, as this is a new feature you may have to resync with `rocketpool service resync-eth1` before this takes effect.%s\n", colorYellow, colorReset)
-		}
+	// Print the appropriate warnings before pruning
+	if selectedEc == cfgtypes.ExecutionClient_Geth {
```

Comment:
nb: i removed besu from here because "pruning" it takes a few seconds now, so there's no reason to print the whole "blah blah blah you will miss attestations" stuff.